### ISSUE TITLE
Upgrade pinax-blog to 5.0.0

### ISSUE THREAD

**Issue by paltman** [djangocon/2016.djangocon.us#61]
This releases gives the ability to publish blog posts in the future by setting the `published` in the admin to a future date.

*Backwards Incompatibility*: URLs in `pinax-blog` are now namespaced to `pinax_blog` so any templates or other code reversing `pinax-blog` urls will need to prefix them with `pinax_blog:` also, in the `urls.py` of the project, the `include("pinax.blog.urls")` will need to change to `include("pinax.blog.urls", namespace="pinax_blog")`.

**Comment by jefftriplett** [comment]
Thanks for the update. 

I updated to a branch and I wasn't seeing anything obvious but it's late. I'll look tomorrow.

```
NoReverseMatch at /blog/feed/all/atom/
Reverse for '('pinax_blog:blog_feed',)' with arguments '()' and keyword arguments '{'section': 'all', 'feed_type': u'atom'}' not found. 0 pattern(s) tried: []
```

I think this is carry over too from the older code. I noticed pinax-account checked for `htttps`: https://github.com/pinax/pinax-blog/blob/master/pinax/blog/views.py#L171

I'm happy to file a ticket tomorrow about updating the link (might cause an SSL error but I haven't gotten that far).

**Comment by jefftriplett** [comment]
Looks like the Atom feed is working now. Thanks again!

**Comment by paltman** [comment]
Excellent news!

On Thu, Mar 24, 2016 at 2:05 PM, Jeff Triplett <notifications@github.com>
wrote:

> Closed #61 <https://github.com/djangocon/2016.djangocon.us/issues/61>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/djangocon/2016.djangocon.us/issues/61#event-602125700>
>



-- 
Patrick Altman
Nashville, TN

http://paltman.com
